### PR TITLE
Change SEO medata Description for BBC Azeri

### LINKED ISSUE #9514: Change SEO medata Description for BBC Azeri
**Is your feature request related to a problem? Please describe.**
The Azeri Service want to update the SEO meta description for the service to improve their SEO

They want to change it from 

"BBC News Azərbaycanca öz səhifələrində Azərbaycan və dünyadan bu günün xəbərləri və şərhlərini təqdim edir. Visit BBC News Azeri for the latest from Azerbaijan, Armenia, Georgia."

image

To

"BBC News Azərbaycanca öz səhifələrində Azərbaycan və dünyadan bu günün xəbərləri və şərhlərini təqdim edir. Visit BBC News Azerbaijan for the latest from Azerbaijan, Armenia, Georgia."

Users are more likely to search for BBC Azerbaijan rather than BBC Azeri. 

**Describe the solution you'd like**
Update the config to reflect the change requested above. 

https://github.com/bbc/simorgh/blob/be9497217bb82e2b1c80e7b7f7d962f5ba901a0a/data/azeri/frontpage/index.json

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by DarioR01**
Resolves #9514 

**Overall change:**
Updated SEO metadata description for BBC Azeri

**Code changes:**

- Replaced text description in Azery hompage metadata to new description

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by jroebu14 — APPROVED** [review]

File: data/azeri/frontpage/index.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
       "feature-main"
     ],
     "title": "Xəbərlər",
-    "summary": "BBC News Azərbaycanca öz səhifələrində Azərbaycan və dünyadan bu günün xəbərləri və şərhlərini təqdim edir. Visit BBC News Azeri for the latest from Azerbaijan, Armenia, Georgia."
+    "summary": "BBC News Azərbaycanca öz səhifələrində Azərbaycan və dünyadan bu günün xəbərləri və şərhlərini təqdim edir. Visit BBC News BBC News Azerbaijan for the latest from Azerbaijan, Armenia, Georgia."
```
Comment (amoore108) [inline data/azeri/frontpage/index.json]: I see this is the text in the linked issue, but should it have a double `BBC News BBC News` before `Azerbaijan`?

**Comment by jimjohnsonrollings** [comment]
Sorry to be a nuisance, but I don't think this actually changes what's wanted by the issue.

This update changes the fixture data, which is only used on localhost.

The production value comes from the Ares feed, and so would need a CPS Config change to update, rather than a code change.

File: data/azeri/frontpage/index.json (hunk 1 above)
Comment (DarioR01) [reply data/azeri/frontpage/index.json]: Yes, it should only have one BBC News, just spoke with Jane about it. Thanks @amoore108

Comment (amoore108) [reply data/azeri/frontpage/index.json]: No worries! 😄

**Comment by aeroplanejane** [comment]
@DarioR01 I've raised a change request to get this data updated in the feeds see CPSSITE-2267
Once this is done (hopefully tomorrow we can close the issue. In the meantime you're free to merge your pr and update the fixture data

**Review by amoore108 — APPROVED** [review]

**Pull request merged**